tests/dashbio_demos/app_needle_plot.py

```diff
@@ -103,108 +103,258 @@ def description():
+                                            ]),
                                         ),
                                         html.Div(
-                                            id='needle-protein-domains-select-div',
-                                            title='If checked, it will allow the user to load '
-                                                  'mutation data such as the protein coordinate '
-                                                  '(x), mutation number (y) and mutation type '
-                                                  '(mutationGroups), individually from the protein'
-                                                  ' domains',
-                                            className='needle-dataset-header-div',
-                                            children=dcc.Checklist(
-                                                id='needle-protein-domains-select-checklist',
-                                                options=[
-                                                    {
-                                                        'label': 'Load protein domains '
-                                                                 'individually',
-                                                        'value': INDIV_DOMS_KEY
-                                                    },
-                                                    {
-                                                        'label': 'Load protein domains from '
-                                                                 'UniProt only',
-                                                        'value': UNIPROT_DOMS_KEY
-                                                    }
-                                                ],
-                                                values=[]
-                                            )
+                                            id='needle-mutdata-file-info-div'
+                                        )
+                                    ]
+                                ),
+                                html.Div(
+                                    id='needle-domain-file-div',
+                                    title='Protein data files accepted here can be two'
+                                          ' fold : \n'
+                                          '- an array of json objects with the required '
+                                          'fields "name" and "coord", which are a string '
+                                          'and a string  detailing the start and end of '
+                                          'the domains in protein coordinate (eg. "23-34")'
+                                          ', respectively.\n'
+                                          '- an json file with the same structure as the '
+                                          'one in the PFAM database : '
```

Comment:
and I would remove this line

Reply:
It might be important to have an example of the structure the JSON file should have, but maybe we should instead link to one of the files we have for example data in this repo.

Reply:
Absolutely! My suggestion is only about the wording: "... same structure as <link>"